.github/workflows/build.yaml

```diff
@@ -0,0 +1,38 @@
+name: build experimental images
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  build_and_push_job:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image:
+        - metadata-webhook
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why do this vs. building the image via the "usual" ci-operator means?

Reply:
Thank you. Yes, we should use the "usual" ci-operator. I added and prepared https://github.com/openshift/release/pull/24094 as well.